hypertrace-trace-enricher/trace-reader/src/main/java/org/hypertrace/trace/accessor/entities/TraceEntityClientContext.java

```diff
@@ -1,4 +1,4 @@
-package org.hypertrace.trace.reader.entities;
+package org.hypertrace.trace.accessor.entities;
```

Comment:
I moved this package, but I left the attribute reader and the project named `trace-reader` to keep the diff from getting too crazy. I'll rename those packages/classes and the project itself later (probably when we start supporting write for attributes, unless there's a desire to do it sooner).